NSS-Bashful-Turtles/Yak-Social-Network
# first commit

Steps to test:
```
git fetch --all
git checkout CDauth
```
1. clear local and session storage
2. input login information.  should redirect to the nav component and everything should work normally.
3. repeat 1 and 2, but register new user instead of logging in
